yang/vyatta-policy-route-v1.yang

```diff
@@ -972,6 +972,13 @@ module vyatta-policy-route-v1 {
 							description "IS-IS level";
 							configd:help "IS-IS level";
 						}
+						leaf src {
```

Comment:
I think `source` would be better as it is not much longer and is consistent with other nodes in this module (eg. access lists).